presentation/src/main/java/dk/nodes/template/presentation/ui/sample/SampleFragment.kt

```diff
@@ -9,7 +9,14 @@ import com.google.android.material.snackbar.Snackbar
 import dk.nodes.template.presentation.R
 import dk.nodes.template.presentation.extensions.observeNonNull
 import dk.nodes.template.presentation.ui.base.BaseFragment
+import dk.nodes.template.presentation.ui.base.BaseViewModel
 import kotlinx.android.synthetic.main.fragment_sample.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.channels.ReceiveChannel
+import kotlinx.coroutines.channels.consumeEach
+import kotlinx.coroutines.channels.produce
+import kotlin.coroutines.CoroutineContext
```

Comment:
Minor thing, but these imports are not used, remember to apply spotless to solve bikeshedding